### PR TITLE
PoC - Vite and Tanstack Router, Query and `@plone/client` in SSR mode.

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: apps/vite/README.md
Comment (stevepiercy) [inline apps/vite/README.md:1]: This file needs to be added to https://github.com/plone/volto/blob/main/.github/workflows/readme-link-check.yml#L23

File: apps/vite/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Example
```
Comment (stevepiercy) [inline apps/vite/README.md:1]:
```suggestion
# Vite

A package for Volto. (I'm sure you can come up with something better.)

- What does this do?
- Who is it for?
- Add link to Vite site.

## Usage
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Docs please!

File: apps/vite/README.md (hunk 1 above)
Comment (sneridagh) [reply apps/vite/README.md]: Not a package, a PoC. It's intended for devs interested in the Plone's headless CMS story or in doing things outside Volto (as all the other things in apps folder)

File: apps/vite-ssr/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Plone on Vite with SSR mode
+
+This is a proof of concept of a [Vite](https://https://vitejs.dev) build, using `@plone/client` and `@plone/components` library.
```
Comment (stevepiercy) [inline apps/vite-ssr/README.md:3]:
```suggestion
This is a proof of concept of a [Vite](https://vitejs.dev) build, using `@plone/client` and `@plone/components` libraries.
```

File: apps/vite-ssr/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+This is intended to serve as both a playground for the development of both packages and as demo of Plone using Vite build with Server Side Rendering (SSR).
```
Comment (stevepiercy) [inline apps/vite-ssr/README.md:4]:
```suggestion
This is intended to serve as both a playground for the development of both packages and as a demo of Plone using Vite built with server side rendering (SSR).
```

File: apps/vite-ssr/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+It also uses TanStack Router as routing library.
```
Comment (stevepiercy) [inline apps/vite-ssr/README.md:6]:
```suggestion
It also uses [TanStack Router](https://tanstack.com/router/latest/docs/framework/react/overview) for its routing library.
```

**Review by stevepiercy — DISMISSED** [review]
Thanks, minor grammar fixes and a link fix.

**Review by stevepiercy — COMMENTED** [review]
README and CI looks good. Thank you!

**Review by stevepiercy — COMMENTED** [review]
Nice catch on nextjs/README.md. CI and README looks good.

File: apps/nextjs/README.md
Comment (stevepiercy) [inline apps/nextjs/README.md:1]: Line 24 fails. The bot incorrectly parses a literal, when it should ignore it. So this workaround might take care of it:
````md
You need to create this environment variable in the Vercel deployment's control panel, specifying the URL where your backend API server is deployed and the route where the API is located as shown.

```shell
API_SERVER_URL=https://my_server_DNS_name/api
```
````

**Review by stevepiercy — DISMISSED** [review]
Try this to workaround the readme check failure.

If it works, then you can remove the whitelist item in the CI that you added in an attempt to get it to pass.

**Comment by sneridagh** [comment]
@stevepiercy it still fails...

**Comment by stevepiercy** [comment]
oh lol `_` is not a legal character in a domain name, hence the failure.

**Comment by stevepiercy** [comment]
I learned to Internet this many years ago 0️⃣ . lol. All right, that readme is good to go now. Just needs a change log and update branch. Back to you @sneridagh.

**Pull request merged**